packages/sources/src/components/label/label.stories.ts

```diff
@@ -1,12 +1,15 @@
 import { action } from '@storybook/addon-actions';
+import { PartialStoryFn } from '@storybook/addons';
 
 import { bindTemplate, StorybookParameters } from '../../stories-helpers';
 
 import { labelArgsMapper, labelArgTypes } from './label.args';
+import { css } from './label.demo';
 import { Label } from './label.models';
 
 export interface LabelParameters<TemplateFnReturnType> {
   labelTemplate: (labelProperties: Label) => TemplateFnReturnType;
+  decorator: (story: PartialStoryFn<any>, css: string) => TemplateFnReturnType;
```

Comment:
- [ ] Bij het mergen kijken of `<any>` vervangen kan worden met `<TemplateFnReturnType>`.